prow/githuboauth/githuboauth.go

```diff
@@ -135,7 +162,7 @@ func (ga *Agent) HandleLogout(client OAuthClient) http.HandlerFunc {
 			loginCookie.Expires = time.Now().Add(-time.Hour * 24)
 			http.SetCookie(w, loginCookie)
 		}
-		http.Redirect(w, r, ga.gc.FinalRedirectURL, http.StatusFound)
+		http.Redirect(w, r, r.URL.Host, http.StatusFound)
```

Comment:
@Katharine is using URL.Host a security issue?

Reply:
Nope. A malicious actor can in theory change the value to confuse us, but they can't change it for someone else under any circumstances, so in this context the worst they can do is mis-redirect themselves.

(I haven't read anything else in this code, just answering this question right now.)